tests/modules/lint/test_main_nf.py

```diff
@@ -97,10 +97,14 @@ class TestMainNfLinting(TestModules):
     - General module linting workflow
     """
 
-    def test_modules_lint_registry(self):
-        """Test linting the samtools module and alternative registry"""
+    def setUp(self):
+        """Set up test fixtures by installing required modules"""
+        super().setUp()
+        # Install samtools/sort module for all tests in this class
         assert self.mods_install.install("samtools/sort")
```

Comment:
I don't think an assert in a setUp step looks weird, but maybe that's an accepted standard?